tutoraspects/templates/aspects/apps/superset/conf/README.md

```diff
@@ -2,6 +2,6 @@
 
 These files are managed via the OEP-58 process and should not be manually edited as they will be overwritten. The process is:
 
-1. A Github action on `openedx-translations` will pull this repo and run `make extract-translations` on this repo's Makefile. This pulls all translatable strings from the Superset assets. These are then uploaded to Transifex for translation.
+1. A Github action on `openedx-translations` will pull this repo and run `make extract_translations` on this repo's Makefile. This pulls all translatable strings from the Superset assets. These are then uploaded to Transifex for translation.
 2. As translations are updated in Transifex they will be pushed to the correct language's files in `openedx-translations`.
 3. A Github action on this repo will run `make pull-translations` periodically. This will run the Atlas tool to synchronize the translation files in this directory with what is stored in `openedx-translations`. Then it will run a command to compile all of the separate translated files into one `locale.yaml` that is used to create the localized dashboard and chart strings during the Tutor init command. It will then automatically merge those changed files.
```

Comment:
This is actually no longer true, translations are actually pulled from Transifex using Atlas at _image build time_. Kind of unrelated to your changes, so if you want to punt on it for this PR that's fine. I can make a separate ticket for it.

Reply:
we can just do it in a separate ticket!